plugins/dev-tools/src/populateRandom.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Helper for randomly populating a workspace.
+ * @author fenichel@google.com (Rachel Fenichel)
+ */
+import Blockly from 'blockly/core';
+
+/**
+ * Populate the workspace with a random set of blocks, for testing.
+ * @param {Blockly.Workspace} workspace The workspace to populate.
+ * @param {number} count How many blocks to create.
+ */
+export function populateRandom(workspace, count) {
+  const names = [];
+  for (const blockName in Blockly.Blocks) {
+    if (Object.prototype.hasOwnProperty.call(Blockly.Blocks, blockName)) {
```

Comment:
For in + hasOwnProperty is a good pattern, so +1. However, why are we calling hasOwnProperty like this? ie: why not:
```
Object.hasOwnProperty(...)
```

Reply:
hasOwnProperty is forced by this: https://eslint.org/docs/rules/guard-for-in
This style of call is forced by https://eslint.org/docs/rules/no-prototype-builtins

We could also just turn off the latter rule. Thoughts?

Reply:
Ah, yeah we should fix that on the lint side, let me look into that.